interface/super/edit_globals.php

```diff
@@ -372,7 +373,7 @@ function checkBackgroundServices()
                     </div>
                     <br />
                     <div id="globals-div">
-                        <ul class="tabNav tabWidthWide" id="oe-nav-ul">
+                        <ul class="tabNav tabWidthWide sticky-top" id="oe-nav-ul">
                         <?php
                         $i = 0;
                         foreach ($GLOBALS_METADATA as $grpname => $grparr) {
```

Comment:
Hi @tywrenn 

I forgot to ask you, do you have any plan to convert the tab to the Bootstrap tab component?